src/lib.rs

```diff
@@ -86,6 +86,7 @@ pub use self::{
     pause::PausedState,
     quic_p2p::Config as TransportConfig,
     quic_p2p::Event as TransportEvent,
+    section::SectionProofChain,
```

Comment:
Need to also remove `SectionProofChain` from the `[cfg(feature = "mock_base")]` block below (line 115)